README.md

```diff
@@ -5,14 +5,28 @@ Docusaurus website serving to showcase the AI Lab's activities.
 https://ai-cfia.github.io/ai-cfia-ia-acia.github.io/
 
 ## Developing Locally
-After opening the project in the devcontainer, change your current working directory:  
+After opening the project in the devcontainer, change your current working directory:
 
 `cd ai-cfia-ia-acia`
 
 Then install the npm dependencies:
 
 `npm install`
 
-You can now run the development server: 
+You can now run the development server:
 
-`npm start -- --port 3001`
\ No newline at end of file
+`npm start -- --port 3001`
+
+## Deployment
+This website is automatically deployed to GitHub Pages when changes are pushed to the `main` branch. The deployment process is managed through GitHub Actions workflows.
+
+### Deployment Workflow
+When changes are pushed to the `main` branch, the following actions are taken by the GitHub Actions workflow:
+
+1. **Trigger**: The `Deploy to GitHub Pages` workflow is triggered on every push to the `main` branch. During the workflow run, static assets are built and then moved to the `deployment` branch, from which the website is served.
```

Comment:
you're repeating three times in this doc that "changes are pushed to the main branch", very redundant.

Reply:
no, I want you to rework this document so that you only need to mention it once, as the trigger and then the rest of the flow follows.

Reply:
done